snapmaker/src/module/toolhead_laser.cpp

```diff
@@ -234,10 +235,15 @@ void ToolHeadLaser::TryCloseFan() {
 
 ErrCode ToolHeadLaser::LoadFocus() {
   CanStdMesgCmd_t cmd;
-
+  uint8_t data[1];
+  if (rotaryModule.status() != ROTATE_OFFLINE) {
+    data[0] = 1;
+  } else {
+    data[0] = 0;
+  }
   cmd.id     = msg_id_get_focus_;
-  cmd.length = 0;
-
+  cmd.length = 1;
```

Comment:
useing sizeof data would be better

Reply:
I don't think so, sometimes the length of buffer array will be larger then the used length